index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header-section">
+        <a href="" class="logo"> My logo </a>
+<nav class="bttn">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+</nav>
+    </header>
+    <section  >
+        <img class="background-image"src="i-bttn/background-shapes/header-shapes mobile.png" alt="">
+        <p class="text"> <h1>I’m Trevor <br> Glad to see you!</h1></p> 
```

Comment:
- [ ] You should stick to the design as much as possible (e.g. font, colors, images, text, margins), Kindly make sure you update the font of this `h1` tag to stick to the Figma design
Your design
![Screenshot from 2023-01-11 11-25-54](https://user-images.githubusercontent.com/80458199/211768869-d9443038-6c76-4d06-aa51-09236cff8167.png)

Figma Design

![Screenshot from 2023-01-11 11-26-18](https://user-images.githubusercontent.com/80458199/211768959-35b24f5e-a18b-42c2-a474-9240e9a868fa.png)